superbench/benchmarks/micro_benchmarks/gpu_copy_performance/gpu_copy.cu

```diff
@@ -797,8 +795,8 @@ int main(int argc, char **argv) {
                     args_list.push_back(args);
                 }
             }
-            // Device-to-host benchmark
-            if (opts.dtoh_enabled) {
+            // Device-to-host benchmark, only used in unidirectional mode.
+            if (opts.dtoh_enabled && !opts.bidirectional_enabled) {
```

Comment:
Question: What's the meaning of Remove bidirectional self-to-self tests because unidirectional self-to-self tests are sufficient? I think it's GPU memory bandwidth test? If so, I think both table can have it.

Reply:
Yes, it’s GPU memory bandwidth test, where uni test launches 1 kernel and bi test launches 2 kernels simultaneously. I think 1 kernel is sufficient to get GPU memory bandwidth?

Reply:
But I can disable uni-directional or bi-directional test, it should be kept.

Reply:
I'll add it back. BTW, actually this modification is not related to self-to-self test. I'll add it in another place.

Reply:
Fixed, please check again, thanks!